src/routes/first-time-configuration-route.php

```diff
@@ -0,0 +1,411 @@
+			'methods'             => 'POST',
+			'callback'            => [ $this, 'set_social_profiles' ],
+			'permission_callback' => [ $this, 'can_manage_options' ],
+			'args'                => [
+				'facebook_site' => [
+					'type'     => 'string',
+				],
+				'twitter_site' => [
+					'type'     => 'string',
+				],
+				'other_social_urls' => [
+					'type'     => 'array',
+				],
+			],
+		];
+		\register_rest_route( Main::API_V1_NAMESPACE, self::CONFIGURATION_ROUTE . self::SOCIAL_PROFILES_ROUTE, $social_profiles_route );
+
+		$person_social_profiles_route = [
+			[
+				'methods'             => 'GET',
+				'callback'            => [ $this, 'get_person_social_profiles' ],
+				'permission_callback' => [ $this, 'can_manage_options' ],
+				'args'                => [
+					'user_id' => [
+						'required' => true,
+					],
+				],
+			],
+			[
+				'methods'             => 'POST',
+				'callback'            => [ $this, 'set_person_social_profiles' ],
+				'permission_callback' => [ $this, 'can_edit_user' ],
+				'args'                => [
+					'user_id' => [
+						'type'     => 'integer',
+					],
+					'facebook' => [
+						'type'     => 'string',
+					],
+					'instagram' => [
+						'type'     => 'string',
+					],
+					'linkedin' => [
+						'type'     => 'string',
+					],
+					'myspace' => [
+						'type'     => 'string',
+					],
+					'pinterest' => [
+						'type'     => 'string',
+					],
+					'soundcloud' => [
+						'type'     => 'string',
+					],
+					'tumblr' => [
+						'type'     => 'string',
+					],
+					'twitter' => [
+						'type'     => 'string',
+					],
+					'youtube' => [
+						'type'     => 'string',
+					],
+					'wikipedia' => [
+						'type'     => 'string',
+					],
+				],
+			],
+		];
+		\register_rest_route( Main::API_V1_NAMESPACE, self::CONFIGURATION_ROUTE . self::PERSON_SOCIAL_PROFILES_ROUTE, $person_social_profiles_route );
+
+		$check_capability_route = [
+			'methods'             => 'GET',
+			'callback'            => [ $this, 'check_capability' ],
+			'permission_callback' => [ $this, 'can_manage_options' ],
+			'args'                => [
+				'user_id' => [
+					'required' => true,
+				],
+			],
+		];
+		\register_rest_route( Main::API_V1_NAMESPACE, self::CONFIGURATION_ROUTE . self::CHECK_CAPABILITY_ROUTE, $check_capability_route );
+
+		$enable_tracking_route = [
+			'methods'             => 'POST',
+			'callback'            => [ $this, 'set_enable_tracking' ],
+			'permission_callback' => [ $this, 'can_manage_options' ],
+			'args'                => [
+				'tracking' => [
+					'type'     => 'boolean',
+					'required' => true,
+				],
+			],
+		];
+		\register_rest_route( Main::API_V1_NAMESPACE, self::CONFIGURATION_ROUTE . self::ENABLE_TRACKING_ROUTE, $enable_tracking_route );
+
+		$save_configuration_state_route = [
+			'methods'             => 'POST',
+			'callback'            => [ $this, 'save_configuration_state' ],
+			'permission_callback' => [ $this, 'can_edit_other_posts' ],
```

Comment:
Is there a reason we don't have `can_manage_options` as a required capability here?

Reply:
Not really, I just used the wrong one 😅

Reply:
Haha ok 😅